src/legacy/core_plugins/data/public/index_patterns/index_patterns/index_patterns.test.ts

```diff
@@ -65,14 +64,14 @@ describe('IndexPatterns', () => {
     const savedObjectsClient = {} as SavedObjectsClientContract;
     const uiSettings = {} as UiSettingsClientContract;
     const http = {} as HttpServiceBase;
-    const notifications = {} as NotificationsSetup;
 
-    indexPatterns = new IndexPatterns(uiSettings, savedObjectsClient, http, notifications);
+    indexPatterns = new IndexPatterns(uiSettings, savedObjectsClient, http);
   });
 
-  test('does cache gets for the same id', () => {
+  test('does cache gets for the same id', async () => {
     const id = '1';
+    const indexPattern = await indexPatterns.get(id);
 
-    expect(indexPatterns.get(id)).toBe(indexPatterns.get(id));
+    expect(indexPattern).toBeDefined();
```

Comment:
this does not test if cache works, please revert to the old test which checks if indexPatterns returns the same object if called twice with same id.